JSDemos/Demos/DataGrid/ExcelJSExportImages/Angular/app/app.component.ts

```diff
@@ -47,7 +47,9 @@ export class AppComponent {
 
             worksheet.getRow(excelCell.row).height = 90;
             worksheet.addImage(image, {
```

Comment:
Take a look at [this comment](https://github.com/DevExpress/devextreme-demos/pull/2684#discussion_r1262414111) in the [adjacent PR](https://github.com/DevExpress/devextreme-demos/pull/2684)